component/org.wso2.carbon.identity.api.otp.service.smsOtp/src/gen/java/org/wso2/identity/api/otp/service/smsotp/SmsotpApi.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year is wrong. Fix here and everywhere else.

Reply:
fixed in c9d7f973185aeb0a97a4a6fce7b36da962169a21